src/Radar.js

```diff
@@ -0,0 +1,237 @@
+/**
+    @external Viz
+    @see https://github.com/d3plus/d3plus-viz#Viz
+*/
+import {nest} from "d3-collection";
+import * as d3 from "d3-selection";
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import {Circle, Path} from "d3plus-shape";
+import {TextBox} from "d3plus-text";
+
+import {default as Plot} from "./Plot";
+
+const tau = Math.PI * 2;
+
+/**
+    @class Radar
+    @extends Plot
+    @desc Creates a radar visualization based on an array of data.
+*/
+export default class Radar extends Plot {
```

Comment:
you should not be extending `Plot`, just extend `Viz`

`Plot` is only for x/y coordinate charts.